examples/with-stomp/useClient.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Stomp } from '@stomp/stompjs'
+
+let stompClient
+
+const useClient = () => {
+  const [client, setClient] = React.useState(stompClient)
+
+  React.useEffect(() => {
+    if (!stompClient) {
+      stompClient = Stomp.client(process.env.STOMP_SERVER)
```

Comment:
This is the last suggestion and the PR is ready, first add this to `next.config.js` like this:

```
module.exports = {
  env: {
    STOMP_SERVER: process.env.STOMP_SERVER
  }
}
```

And then update the readme to include a step for this too, because I assume that without that env the example won't work. Bonus points if you can provide a public one

Reply:
@lfades Thank you, Luis, I made some changes.